resp/resp3/resp.go

```diff
@@ -2004,6 +2011,17 @@ func unmarshalAgg(prefix Prefix, br resp.BufferedReader, l int64, rcv interface{
 		l *= 2
 	}
 
+	if o == nil || o.TreatErrorsAsValues == false {
```

Comment:
I don't think opts should ever be nil, so need to check for that.

Reply:
I just assumed it was ok for it to be nil, but it looks like all the code assumes it's non-nil, so I removed the check.